README.md

```diff
@@ -54,6 +54,13 @@ agents {
         bootstrap.servers = kafka-svc:9092
         producer.topic = spans
       }
+      
+      http {
+        url = http://collector-svc:8080/spans
+        callTimeoutMillis = 500
+        connectionPool.maxIdleConnections = 5
+        connectionPool.keepAliveDurationMinutes = 5
```

Comment:
We try to follow lowercase convention for config keys. It helps us to override them through env variable.
For e.g. HAYSTACK_ENV_HTTP_CONNNECTIONPOOL_MAXIDLECONNECTIONS.

Reply:
Got it. The Kafka and the Kinesis are inconsistent, should I change the ones for Kinesis as well?

Reply:
For kafka, it should all be lower case. In case you find it inconsistent, please do. 
For kinesis, we use reflection to configure kinesis-producer. Should we do the same approach for  okhttp client?
https://github.com/ExpediaDotCom/haystack-agent/blob/master/agent-dispatchers/kinesis/src/main/java/com/expedia/www/haystack/agent/dispatcher/KinesisDispatcher.java#L218

we try to avoid adding a layer on top of native properties. We follow this model for both kafka and kinesis dispatchers

Reply:
I made them all lower case.